src/include/nodes/parsenodes.h

```diff
@@ -4260,4 +4260,15 @@ typedef struct RetrieveStmt
 	bool		is_all;
 } RetrieveStmt;
 
+/* ----------------------
+ *		Warehouse Statement
+ * ----------------------
+ */
+typedef struct CreateWarehouseStmt
```

Comment:
Add _copyCreateWarehouseStmt、_equalCreateWarehouseStmt and _outCreateWarehouseStmt(if needed) for CreateWarehouseStmt?

Reply:
The `WarehouseStmt` will not be dispatched to Segments, I think it is not necessary.(Or we can add it when we need it)